src/app/store.ts

```diff
@@ -1,10 +1,17 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import statement for `combineSlices` is incorrect. You should import `combineReducers` from `@reduxjs/toolkit` instead. This function is used to combine multiple slice reducers into a single root reducer.